package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "posthog-js",
-    "version": "1.7.1",
+    "version": "1.7.2",
```

Comment:
? weird - not sure why this + featureflags.js tests are showing up here.

Maybe worth rebasing?

Reply:
Soz, hadn't pushed this from my hotfix this weekend. This is now in master and rebased. Feature flag is a legit small cleanup